config/initializers/storyblok_client.rb

```diff
@@ -1,18 +1,24 @@
 storyblok_token = ENV.fetch('STORYBLOK_TOKEN') { raise 'STORYBLOK_TOKEN missing' }
 
+logger = Rails.env.development? ? Rails.logger : nil
+
 STORYBLOK_CLIENT = if ENV['CONTENT_PREVIEW_MODE'] == 'true'
                      Storyblok::Client.new(
                        token: storyblok_token,
+                       logger: logger,
                        version: 'draft'
                      )
                    else
                      redis_cache_url = ENV.fetch('REDIS_CACHE_URL') { raise 'REDIS_CACHE_URL missing' }
                      redis = Redis.new url: redis_cache_url
-                     cache = Storyblok::Cache::Redis.new redis: redis
+
+                     ttl = ENV.fetch('CONTENT_CACHE_TTL_MINS', 60).to_i * 60
```

Comment:
Add in a raise here

Reply:
Figured this could be an optional env var, and the default of an hour makes sense. So no need to raise an error if it's not set.